shared/src/components/question/answer.cjsx

```diff
@@ -1,5 +1,5 @@
 React = require 'react'
-_ = require 'underscore'
+{partial, pick, debounce} = require 'lodash'
 classnames = require 'classnames'
```

Comment:
This file is still coffeescript so we have to individually import lodash stuffs, i.e.
```
partial = require 'lodash/partial'
...
```

Reply:
oooh right, okay, will change